README.md

```diff
@@ -65,3 +65,34 @@ This project adheres to the [StandardJS Style Guide](https://github.com/standard
 ## Entity-Relationship Diagram
 
 ![](./diagram.png)
+
+## Table Definitions
+
+```sql
+CREATE TABLE users(
+  id SERIAL PRIMARY KEY,
+  username varchar(32) UNIQUE NOT NULL,
+  email varchar(50) UNIQUE NOT NULL,
+  password text NOT NULL
+);
+
+CREATE TABLE communities(
+  id SERIAL PRIMARY KEY,
+  username varchar(32) UNIQUE NOT NULL
```

Comment:
What if communities have just `name` instead of `username`, feels more predictable